charts/argo-workflows/templates/controller/workflow-controller-deployment.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-{{ .Values.controller.name}}
+  labels:
+    app: {{ .Release.Name }}-{{ .Values.controller.name}}
```

Comment:
the deployments are currently using the "old-style" labels while newer deployments/charts generally use the recommended labels such as `app.kubernetes.io/name` (see https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)

Should we update these as well?

Reply:
I was wondering if we should add a library chart that has the same labels/ selectors etc. for all charts. We recently implemented the new-style labels in the argo-cd chart. I would suggest for now we leave it old style but i will create an issue addressing this for all charts.

Reply:
Sounds good!

Reply:
I'd personally like to see these updated to the recommended style. However doing so is a breaking change as selector labels are immutable.

Reply:
You're right! I'm OK with either approach. I've looked at https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/templates/_helpers.tpl and could definitely copy-and-paste them over to this chart for now until we have a library chart available. Since this is the first version of this chart and a 0.* release, most people will need to freshly install anyway.